pybamm/models/submodels/thermal/pouch_cell/pouch_cell_1D_current_collectors.py

```diff
@@ -60,40 +60,82 @@ def set_rhs(self, variables):
         # Account for surface area to volume ratio of pouch cell in cooling
         # coefficient. Note: the factor 1/delta^2 comes from the choice of
         # non-dimensionalisation
-        A = self.param.l_y * self.param.l_z
-        V = self.param.l * self.param.l_y * self.param.l_z
-        cooling_coeff = -2 * self.param.h * A / V / (self.param.delta ** 2)
+        cell_volume = self.param.l * self.param.l_y * self.param.l_z
+
+        yz_surface_area = self.param.l_y * self.param.l_z
+        yz_surface_cooling_coefficient = (
+            -(self.param.h_cn + self.param.h_cp)
+            * yz_surface_area
+            / cell_volume
+            / (self.param.delta ** 2)
+        )
+
+        side_edge_area = 2 * self.param.l_z * self.param.l
+        side_edge_cooling_coefficient = (
+            -self.param.h_edge * side_edge_area / cell_volume / self.param.delta
+        )
+
+        total_cooling_coefficient = (
+            yz_surface_cooling_coefficient + side_edge_cooling_coefficient
+        )
 
         self.rhs = {
             T_av: (
                 pybamm.laplacian(T_av)
                 + self.param.B * Q_av
-                + cooling_coeff * (T_av - T_amb)
+                + total_cooling_coefficient * (T_av - T_amb)
             )
             / (self.param.C_th * self.param.rho)
         }
 
     def set_boundary_conditions(self, variables):
         T_amb = variables["Ambient temperature"]
         T_av = variables["X-averaged cell temperature"]
-        T_av_left = pybamm.boundary_value(T_av, "negative tab")
-        T_av_right = pybamm.boundary_value(T_av, "positive tab")
+        T_av_top = pybamm.boundary_value(T_av, "right")
+        T_av_bottom = pybamm.boundary_value(T_av, "left")
+
+        # Tab cooling only implemented for both tabs at the top.
```

Comment:
is there a way to generalize this by using a linear combination of the two so that the appropriate cooling switches on/off depending on tab location? just a little concerned that this comment is buried in the model and people will use it with tabs in different locations without realizing it isn't implemented

Reply:
I was just thinking you could e.g. edge cooling everywhere then add something like (1-tab_z_coord)*tab_cool _correction (and all the other appropriate combinations to get the right thing) 

yeah seems like a lot of work. is there somewhere more explicit we can warn that this only does tab cooling from the top?